test/runtest.py

```diff
@@ -1120,13 +1120,17 @@ def g(n):
             res = [ray.put(i) for i in range(1000 // n)]
             ray.wait(res, len(res))
 
-        threads = [
-            threading.Thread(target=g, args=(n, ))
-            for n in [1, 5, 10, 100, 1000]
-        ]
+        @ray.remote
+        def multi_thread_worker():
```

Comment:
Thanks for testing this on the worker, however we should also make sure that the driver can use multiple threads.